### PR TITLE
feat: add digitalagenda

### PR DESCRIPTION
**Pull request by danielasubotic**

### CODE REVIEW

File: services/metadata/backend/data/digitalagenda.json [hunk 1]
```diff
@@ -0,0 +1,416 @@
...
+        "secondaryURL": {
+            "__type": "URL",
+            "type": "URL",
+            "url": "https://www.digitalagenda.ch/",
+            "text": "External Project Website"
+        }
+    },
+    "datasets": [
+        {
+            "__id": "http://ns.dasch.swiss/repository#dsp-0848-dataset-000",
```
Comment (RebeccaSigloch2) [inline services/metadata/backend/data/digitalagenda.json:146]: This link does not work

**Review by RebeccaSigloch2 — COMMENTED** [review]
see Comment

File: services/metadata/backend/data/digitalagenda.json (hunk 1 above)
Comment (danielasubotic) [reply services/metadata/backend/data/digitalagenda.json:146]: it's an internal link which is not relevant for the user

**Review by RebeccaSigloch2 — APPROVED** [review]
looks good to me

**Pull request merged**